entrypoint.sh

```diff
@@ -223,15 +223,14 @@ run_nvidia_installer() {
     insmod `find /root/lib/modules -iname ipmi_devintf.ko`
   fi
   pushd "${NVIDIA_INSTALL_DIR_CONTAINER}"
+  IGNORE_MISSING_MODULE_SYMVERS=1 \
   sh "${NVIDIA_INSTALLER_RUNFILE}" \
+    --driver \
     --kernel-source-path="${KERNEL_SRC_DIR}" \
-    --utility-prefix="${NVIDIA_INSTALL_DIR_CONTAINER}" \
-    --opengl-prefix="${NVIDIA_INSTALL_DIR_CONTAINER}" \
-    --x-prefix="${NVIDIA_INSTALL_DIR_CONTAINER}" \
-    --no-install-compat32-libs \
-    --log-file-name="${NVIDIA_INSTALL_DIR_CONTAINER}/nvidia-installer.log" \
-    --silent \
-    --accept-license
+    --no-drm \
```

Comment:
@lsjostro, I'd like your take on whether omitting these three things match what you think was happening before in the earlier versions.

There are also flags to install the CUDA toolkit (`--toolkit` and `--toolkitpath`) and the the CUDA samples (`--samples` and `--samplespath`). I don't happen to need either of those for my purposes, but perhaps we should have options to include them.